modules/tutorials/pages/java-tutorial/first-couchbase-application.adoc

```diff
@@ -0,0 +1,191 @@
+
+In the xref:java-tutorial/install-couchbase-java-sdk.adoc[] tutorial, you created a directory structure to hold your application.
+Create the file `student.java` in the `java` directory as shown below.
+
+include::partial$student-mvn-directory-structure.adoc[tags="location-of-student.java"]
+
+Add the following content to the `ConnectStudent.java` file.
+
+[source, java]
+----
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.Scope;
+
+import java.time.Duration;
+
+public class ConnectStudent {
+
+    public static void main(String[] args) {
+
+        Cluster cluster = Cluster.connect("localhost",
+                "Administrator", "password");    // <.>
+
+        Bucket bucket = cluster.bucket("student-bucket");    // <.>
+
+        bucket.waitUntilReady(Duration.ofSeconds(10));    // <.>
+
+        Scope scope = bucket.scope("art-school-scope");    //<.>
+
+        Collection student_records = scope.collection("student-record-collection");    // <.>
+
+        System.out.println("The name of this collection is " + student_records.name());    // <.>
+
+        cluster.disconnect();    // <.>
+    }
+}
+
+----
+
+If you're familiar with the JDBC API then connecting to a Couchbase server is not that different:
+
+<.> A call to `Cluster.connect` creates a channel to the named server (`localhost` running on your local machine in
+this case), supplying your username and password to authenticate the connection.
+
+<.> The retrieved `cluster` is used to retrieve the bucket set up in xref:install-couchbase-server.adoc[].
+
+<.> {why-waitUntilReady} Once the function has finished, it sends a notification to its caller, which
+will then process the output of the call. +
+Now this usually works very well in practice, but in our simple example there is a problem: after we make the call
+to `cluster.bucket`, the program will carry on without waiting for the bucket retrieval to be complete. This means
+we're now trying to retrieve the scope from a bucket object that hasn't been fully initialised. +
+This is where the `waitUntilReady` call comes in. This will force the program to wait until the bucket object is
+fully prepared.
+
+<.> Retrieves the `art-school-scope` from the bucket.
+
+<.> Retrieves the student collection from the scope.
+
+<.> The next line is just a check to make sure we have connected and retrieved the collection when the program is run.
+You can see the output by running it through maven:
+
+<.> And as with all database systems, it's good practise to disconnect from the Couchbase cluster once you've
+finished working with it.
+
+[source, bash]
+----
+mvn exec:java -Dexec.mainClass="ConnectStudent" -Dexec.cleanupDaemonThreads=false
+----
+
+Somewhere in the output, you'll find the line containing the name of the collection you've successfully retrieved:
+
+image::connect-to-cluster-console-output.png[]
```

Comment:
nice visualisation! I'd make the callout text still bigger though, perhaps remove some of the surrounding text to make space.